Philips.CodeAnalysis.Common/DocumentationHelper.cs

```diff
@@ -87,7 +85,7 @@ public DocumentationCommentTriviaSyntax CreateDocumentation()
 
 		private static bool IsCommentTrivia(SyntaxTrivia trivia)
 		{
-			return trivia.IsKind(SyntaxKind.SingleLineDocumentationCommentTrivia);
+			return trivia.IsKind(SyntaxKind.SingleLineDocumentationCommentTrivia) || trivia.IsKind(SyntaxKind.SingleLineCommentTrivia);
```

Comment:
Maybe good to add a comment to remove this temp fix. I guess this might cause some false positives